savageRoller.ino

```diff
@@ -264,8 +267,9 @@ void rollerHandleKeys() {
     }
     if (isNewlyPressed('b')) {
       // Show the battery screen.
-      firstRun = 1;
       currentPage = Page::Battery;
+      last_battery_check = 0;
+      batteryHandleDisplay();
```

Comment:
Why move this here? Why not just do the state change and let the main loop call the right function?

Reply:
Testing and forgot to put it back? :)